website/src/blogs/2017-02-20-torus-from-coreos-steps-aside-as-cloud-native-storage-platform-what-now.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Torus (from CoreOS) steps aside as Cloud Native Storage Platform. What now?
+author: Kiran Mova
+author_info: Contributor and Maintainer OpenEBS projects. Chief Architect MayaData. Kiran leads overall architecture & is responsible for architecting, solution design & customer adoption of OpenEBS.
+tags: DevOps, Docker, Storage Containers, Kubernetes
+date: 20-02-2017
+excerpt: Torus (from CoreOS), that aimed at providing container-native distributed storage announced that it would discontinue development due to lack of traction.
+---
+
+[Torus (from CoreOS)](https://github.com/coreos/torus), which aimed at providing container-native distributed storage announced that it would discontinue development due to lack of traction. It just goes to show the dynamics of the OpenSource projects and how business needs could influence them.
```

Comment:
```suggestion
[Torus (from CoreOS)](https://github.com/coreos/torus), which aimed at providing container-native distributed storage announced that it would discontinue development due to lack of traction. It just goes to show the dynamics of the Open Source projects and how business needs could influence them.
```